JMS/README.md

```diff
@@ -225,6 +225,16 @@ If you have used maven to build the samples, you can run
 
 The response sample will get a message from the queue, process it and put the response on the reply to queue and keep looking for more messages to respond to till you ctrl+c interrupt it.
 
+### Poison Messages
+While running the request/response samples, if an error, such as "Reply to Queue no longer exists, skipping request" or "Reply to destination is invalid", is received then the message might be a Poison Message. This occurs when the message cannot get processed by the receiving application and gets sent back to the queue. The message will sit in the queue and the application will keep on seeing it until it's removed. This will become an endless loop.
+
+To solve this issue, we can put the message into a backout queue by configuring the queue to detect if there have been more than a set threshold number of attempts to read the message. The last resort is moving the message to a Dead Letter Queue.
```

Comment:
While there is an IBM MQ queue configuration for this, the code / dev should also be defensive i.e., try a number of times and then decide to blackout. That might be another queue or it might be that log something for an admin. The point is that some kind of strategy is required. A blackout queue is an example.